src/player.c

```diff
@@ -1010,8 +1017,33 @@ void player_render( double dt )
     * Render the player.
     */
    if ((player.p != NULL) && !player_isFlag(PLAYER_CREATING) &&
-         !pilot_isFlag( player.p, PILOT_INVISIBLE))
+         !pilot_isFlag( player.p, PILOT_INVISIBLE)) {
+
+      /* Render the aiming lines. */
+      a = player.p->solid->dir;
+      r = 200.;
```

Comment:
I can't think of why at the moment, but the line seems to get shorter the closer the camera is zoomed in (when it should get longer, or stay the same). You can see this clearly if you enable "manual zoom control".